docs/src/app/components/pages/components/Table/ExampleSimple.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import Table from 'material-ui/lib/table/table';
+import TableHeaderColumn from 'material-ui/lib/table/table-header-column';
+import TableRow from 'material-ui/lib/table/table-row';
+import TableHeader from 'material-ui/lib/table/table-header';
+import TableRowColumn from 'material-ui/lib/table/table-row-column';
+import TableBody from 'material-ui/lib/table/table-body';
+import TableFooter from 'material-ui/lib/table/table-footer';
+import TextField from 'material-ui/lib/text-field';
+import Toggle from 'material-ui/lib/toggle';
+
+const propContainerStyle = {
+  width: '200px',
+  overflow: 'hidden',
+  margin: '20px auto 0 auto',
+};
+
+export default class TableExampleSimple extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this._onToggle = this._onToggle.bind(this);
+    this._onChange = this._onChange.bind(this);
+
+    this.state = {
+      fixedHeader: true,
+      fixedFooter: true,
+      stripedRows: false,
+      showRowHover: false,
+      selectable: true,
+      multiSelectable: false,
+      enableSelectAll: false,
+      deselectOnClickaway: true,
+      height: '300px',
+    };
+  }
+
+  _onChange(e) {
+    this.setState({height: e.target.value});
+  }
+
+  _onToggle(e, toggled) {
```

Comment:
What about following https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md?
Hence, we would use `handleToggle`